integrated_channels/cornerstone/views.py

```diff
@@ -12,6 +12,112 @@
 from integrated_channels.cornerstone.models import CornerstoneEnterpriseCustomerConfiguration
 
 
+class BaseViewSet(generics.ListAPIView):
```

Comment:
You say this, but this isn't used for class CornerstoneCoursesListView(generics.ListAPIView). Is that an oversight?